ground/src/main/java/com/google/android/ground/MainActivity.kt

```diff
@@ -190,7 +190,11 @@ class MainActivity : AbstractActivity() {
 
   private fun navigateUp(): Boolean {
     val navController = navHostFragment.navController
-    return navController.navigateUp()
+    return if (navController.currentBackStackEntry != null) {
```

Comment:
Should we use `navHostFragment.childFragmentManager.backStackEntryCount` instead?